update docs for team admin permissions++ (#5564)
pulumi/docs: content/docs/intro/console/accounts-and-organizations/organizations.md

### GitHub-backed

> To add a GitHub-backed organization to Pulumi, an admin of the GitHub organization

Reviewer: Little thing, but since this PR is about changing Console terminology and these words instead refer to GitHub, GitLab and Atlassian "admin-type things", we should probably make sure we're using the right terminology for each provider, just to avoid any confusion.

Author: on very quick google search they all call them admins

associated with, and lets you add a new organization backed by a third
party identity provider. To switch to a different organization:

> 1. Select the organization drop-down list on the upper left corner of the Console screen

Reviewer: ```suggestion
1. Select the organization drop-down list in the upper left corner of the Console screen
```

next to the Pulumi logo.
1. Select your organization name. The organization page view is dependent on your role within the Pulumi organization.

<img class="lg:max-w-xl" src="/images/docs/reference/service/organization-view.png" alt="Stack Outputs and Configuration">

  _Members_. Pulumi organization members only see the Stacks and People tabs.

  _Admins_. Pulumi organization admins see the Stacks, People, Teams, Webhooks,
and Settings tabs.

| Console Tab | Description |
|--------|--------|

> | Dashboard | An overview of the organization including recently updated stacks, recent activiity, and a resource count graph. |

Reviewer: ```suggestion
| Dashboard | An overview of the organization including recently updated stacks, recent activity, and a resource count graph. |
```

> | Projects | A searchable list of organization stacks that you can group by project and tag. For more information, see [Project and Stack Management]({{< relref "/docs/intro/console/collaboration/project-and-stack-management">}}). |

Reviewer: ```suggestion
| Projects | A searchable list of organization stacks that you can group by project and tag. For more information, see [Project and Stack Management]({{< relref "/docs/intro/console/collaboration/project-and-stack-management" >}}). |
```

| Members | A list of active members of the Pulumi organization. |
| Teams | A [Team Pro]({{< relref "/pricing" >}}) feature that provides a way to assign stack permissions to groups of organization members. |

> | Policies | Policies empower you to set guardrails to enforce best practices and compliance. |

Reviewer: This one's a bit weird in that the others explain what the tab is, whereas this one explains the concept of policies. Seems like it's missing something before it, like

```suggestion
| Policies | Lists of organization policies and policy groups. Policies allow you to set guardrails to enforce best practices and compliance. |
```

Author: heh yah thx for fixing it, i just pulled that text from the ui

> | Settings | A place for you to manage your organization settings including subscription and payment information and history, stack permissions, and links to Pulumi's [continuous delivery guides]({{< relref "/docs/guides/continuous-delivery" >}}). |

Reviewer: ```suggestion
| Settings | Organization settings including subscription and payment information and history, stack permissions, and links to Pulumi's [continuous delivery guides]({{< relref "/docs/guides/continuous-delivery" >}}). |
```

| Console Tab | Description |
|--------|--------|
| Dashboard | An overview of the organization including, recently updated stacks, recent activiity, a resource count graph. |
| Projects | A searchable list of organization stacks that you can group by project and tag. See [Project and Stack Management]({{< relref "/docs/intro/console/collaboration/project-and-stack-management">}}) to learn more. |
| Members | A list of active members of the Pulumi organization. |
| Teams | A [Team Pro]({{< relref "/pricing" >}}) feature that provides a way to assign stack permissions to groups of organization members. |
| Policies | Policies empower you to set guardrails to enforce best practices and compliance. |

> | Settings | A place for you to manage your organization settings including subscription and payment information and history, stack permissiongs, and links to Pulumi's [continuous delivery guides]({{< relref "/docs/guides/continuous-delivery" >}}). |

Author: ```suggestion
| Settings | A place for you to manage your organization settings including subscription and payment information and history, stack permissions, and links to Pulumi's [continuous delivery guides]({{< relref "/docs/guides/continuous-delivery" >}}). |
```
